Master

tolits123 opened the pull request.

ashlrem reviewed (approved):
Reviewed, Fix for reciept not getting values from Add Payment Form, and validation if input is greater than total tuition fee.

Angelo-SQL committed:
Update [04162017]

Add payment - (Partial Payment) if the partial payment text box is >
total tutition
